modules/core/client/views/subject_setup.client.view.html

```diff
@@ -0,0 +1,84 @@
+<section data-ng-controller="subjectSetupController" id="content">
+    <side-bar></side-bar>
+
+    <div top="20%">
+        <ul class="breadcrumb" style="width:100%">
+            <li>
+                <a href="/">Home</a>
+            </li>
+            <li class="active">
+                Study #
+            </li>
+        </ul>
+    </div>
+    
+    <table>
+        <tr>
+            <td>Subject ID</td>
+            <td>{{subjectID}}</td>
+        </tr>
+        <tr>
+            <td>Subject Group</td>
+            <td>
+                <div ng-controller="GroupCtrl">
+                   <select name="Title" ng-model="currentItem" class="form-control">
+                    
+                    <option value="">Unassigned</option>
+                    <option ng-repeat= "item in items" value="{{item}}">{{item}}</option>
+                  </select>
+                </div>
+
+            </td>
+        </tr>
+        <tr>
+            <td>Phone Number</td>
+            <td>
+                <div class="input-group">
+                    <!--<span class="input-group-addon" id="basic-addon1">@</span>-->
+                    <input type="text" class="form-control" placeholder="XXX-XXX-XXXX" aria-describedby="basic-addon1">
+                </div>
+            </td>
+        </tr>
+        <tr>
+            <td>Reader ID</td>
+            <td>
+                   <select ng-model="selectedReader" class="form-control">
+                    <option ng-repeat= "reader in readers" value="{{reader}}">{{reader}}</option>
+                  </select>
+            </td>
+        </tr>  
+        <tr>
+            <td>Start Date</td>
+            <td>
+
+                
+                <div ng-controller="DatepickerDemoCtrl">
```

Comment:
!fix Instead of having the controller declared for both of these sections separately just wrap them in a div around the scope of both of these declarations and use that to define the controller. That way it will be able to be scoped there with one declaration!